go/obscuronode/host/host.go

```diff
@@ -368,21 +373,28 @@ func (a *Node) processBlocks(blocks []obscurocommon.EncodedBlock, interrupt *int
 	if result.ProducedRollup.Header != nil {
 		a.P2p.BroadcastRollup(nodecommon.EncodeRollup(result.ProducedRollup.ToRollup()))
 
-		obscurocommon.ScheduleInterrupt(a.cfg.GossipRoundDuration, interrupt, a.handleHeader(result))
+		obscurocommon.ScheduleInterrupt(a.cfg.GossipRoundDuration, interrupt, a.handleRoundWinner(result))
 	}
 }
 
-func (a *Node) handleHeader(result nodecommon.BlockSubmissionResponse) func() {
+func (a *Node) handleRoundWinner(result nodecommon.BlockSubmissionResponse) func() {
+	resultCopy := result
```

Comment:
What does this copy achieve? (For my understanding, not because I think it's wrong)

Reply:
yeah. forgot to remove it. I wanted to make sure that the lambda is somehow working on the right result.